molo/core/tests/test_models.py

```diff
@@ -104,6 +104,12 @@ def test_image(self):
         self.assertEquals(
             new_section3.get_effective_image(), new_section.image)
 
+    def test_number_of_child_sections(self):
+        new_section = self.mk_section(self.section_index)
+        self.mk_sections(new_section, count=12)
+        response = self.client.get('/sections/test-section-0/')
+        self.assertContains(response, 'Test Section 11')
```

Comment:
@miltontony I want the other sections to be there, I want to test that it doesn't stop showing sections after the fifth secion